widgetastic_manageiq/__init__.py

```diff
@@ -3509,19 +3523,6 @@ def entity_class(self):
         else:
             return JSBaseEntity
 
-    @property
```

Comment:
I accidentally put two entity_names properties to one class. One of them has to be removed

Reply:
Good call, the other one is better.